Platform/CommonBoardPkg/Tools/GpioDataConvert.py

```diff
@@ -107,13 +109,21 @@
 
 GPIO_MIXED_INT_CONFIG = {
     'GpioIntLevel|GpioIntApic'  : 0x31,
+    'GpioIntApic|GpioIntLevel'  : 0x31,
     'GpioIntLevel|GpioIntSci'   : 0x29,
+    'GpioIntSci|GpioIntLevel'   : 0x29,
     'GpioIntLevel|GpioIntSmi'   : 0x25,
+    'GpioIntSmi|GpioIntLevel'   : 0x25,
     'GpioIntEdge|GpioIntSci'    : 0x69,
+    'GpioIntSci|GpioIntEdge'    : 0x69,
     'GpioIntEdge|GpioIntApic'   : 0x71,
+    'GpioIntApic|GpioIntEdge'   : 0x71,
     'GpioIntEdge|GpioIntSmi'    : 0x65,
+    'GpioIntSmi|GpioIntEdge'    : 0x65,
     'GpioIntEdge|GpioIntDefault': 0x60,
+    'GpioIntDefault|GpioIntEdge': 0x60,
     'GpioIntLvlEdgDis|GpioIntDis':0xA1,
+    'GpioIntDis|GpioIntLvlEdgDis':0xA1,

```

Comment:
If we keep these ,  I would think better to put script under  Platform/CommonBoardPkg.
GPIO_MIXED_INT_CONFIG is project specific,  and not that common to be in  BootloaderCorePkg.

Reply:
As I said in other comment, I added this only for convenience. Ideally, if there is a mixed config provided in csv/h file, then we need to get the value (from GPIO_INT_CONFIG) for each operand in OR operation and OR the values. This way, we can eliminate GPIO_MIXED_INT_CONFIG. let me add that.